.eslintrc.js

```diff
@@ -18,16 +16,14 @@ module.exports = {
     ecmaVersion: 2018,
     sourceType: 'module',
   },
-  plugins: [
-    'react',
-  ],
+  plugins: ['react'],
   rules: {
-    "import/no-extraneous-dependencies": "off" ,
-    "import/prefer-default-export": "off"
+    'import/no-extraneous-dependencies': 'off',
```

Comment:
Out of curiosity, what command did you run that replaced all the double quotes here with single quotes?

Reply:
I use Prettier as a VS Code extension with the following config:

```
{
  "tabWidth": 2,
  "useTabs": false,
  "singleQuote": true,
  "trailingComma": "all",
  "printWidth": 100
}
```

Reply:
Ahh ok, that explains it!